app/models/update_existing_draft_edition.rb

```diff
@@ -79,4 +95,20 @@ def attributes
   def edition_attributes_from_payload
     payload.slice(*Edition::TOP_LEVEL_FIELDS)
   end
+
+  def publisher_first_published_at
```

Comment:
I don't think we need these methods if we're only using them in one place, I would just take the values straight from the payload.